pkg/openthread/include/openthread/conn.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    pkg_openthread_conn   Connection type definitions for OpenThread
+ * @ingroup     pkg_openthread
+ * @{
+ *
+ * @file
+ *
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+#ifndef OT_CONN_H
+#define OT_CONN_H
+
+#include <openthread.h>
+#include "thread.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ *  * @brief   conn_udp definition for OpenThread
+ *
+ */
+struct conn_udp {
```

Comment:
Yes, `conn_udp` is deprecated for two releases now (though we can't remove it until #6012 is merged ;-))